exonum/src/api/public/system.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2018 The Exonum Team
```

Comment:
Seems like this file is some merge error. New hierarchy doesn't contain `/src/api/public` folder.